src/main/java/com/minecolonies/entity/EntityFishHook.java

```diff
@@ -736,7 +736,6 @@ private void spawnLootAndExp(final EntityCitizen citizen)
      * @param citizen the fisherman getting the loot
      * @return an ItemStack randomly from the loot table
      */
-    @SuppressWarnings("unchecked")
     private ItemStack getFishingLoot(final EntityCitizen citizen)
```

Comment:
We should probably actually check our casts then, so we don't get all the warnings.